test/Elastic.Apm.AspNetFullFramework.Tests/DistributedTracingTests.cs

```diff
@@ -152,6 +152,42 @@ public async Task CallSoapRequest()
 			});
 		}
 
+		[AspNetFullFrameworkFact]
+		public async Task CallSoap12Request()
+		{
+			var rootTxData = SampleAppUrlPaths.CallSoapServiceProtocolV1_2;
+			var fullUrl = Consts.SampleApp.RootUrl + "/" + rootTxData.RelativeUrlPath;
+			var action = "Ping";
+
+			var httpContent = new StringContent($@"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
We need more tests here with more sample data. [Wikipedia](https://en.wikipedia.org/wiki/SOAP) has 1 example, and I'd also add the one from [here](https://www.w3.org/TR/soap12-part1/) (section 1.4 Example SOAP Message).